test/test_web.py

```diff
@@ -141,13 +141,65 @@ def test_main_menu(self) -> None:
         submenu_entry = submenu.find_element(By.XPATH, './/a[contains(@class, "item")]')
         self.assertFalse(submenu_entry.is_displayed())
         submenu.click()
+        actions = ActionChains(self.driver)
+        actions.move_to_element(submenu).perform()  # do a hoover
+
```

Comment:
This looks like simple click events are no longer sufficient to toggle the dropdown menu. This might have negative effects on the usability, especially when using other input devices (like keyboard, touch screen, digitizer pen). As I already commented, we should make sure that these methods of opening the submenu still work without usability limitations. Then, I'm fine with opening the submenus on hover and adapting this test, if still required.